packages/flutter/lib/src/services/assertions.dart

```diff
@@ -124,6 +124,8 @@ class FlutterError extends AssertionError {
 
   static int _errorCount = 0;
 
+  static const int _kWrapWidth = 120;
```

Comment:
I tried 80, but it turns out that looks really freaking narrow and a lot of our messages are substantially wider than that. 120 is what I needed to make the header be roughly as wide as the stack trace on my display. (I don't wrap the stack trace because that would make it unreadable.)